settings/volare-tcp-settings.json

```diff
@@ -0,0 +1,46 @@
+{
+	"vessel": {
+		"name"	: "Volare",
+		"brand"	: "Friendship",
+		"type"	: "22",
+		"uuid"	: "10101010",
+		
+		"dimensions": {
+			"length": 7,
+			"width": 2.5,
+			"mast": 10,
+			"depthTransducer": 0.5,
+			"keel": 1.5 
+		}
+	},
+
+	"interfaces": {
+    "bower": false
+  },
+  
+  "pipedProviders": [{
+    "id": "nmeaFromFile",
+    "pipeElements": [
+       { 
+         "type": "providers/tcp",
+         "options": {
+           "host": "192.168.0.22",
+           "port": "1337"
```

Comment:
You can add here optionMappings to make the host & port configurable/overridable from the command line as well.